### PR TITLE
Remove VS warnings

### PR DESCRIPTION
**Pull request by fbudin69500**

### CODE REVIEW

File: include/itkVariationalRegistrationStopCriterion.hxx [hunk 1]
```diff
@@ -86,7 +86,7 @@ VariationalRegistrationStopCriterion< TRegistrationFilter, TMRFilter >
 template< class TRegistrationFilter, class TMRFilter >
 void
 VariationalRegistrationStopCriterion< TRegistrationFilter, TMRFilter >
-::Execute( const itk::Object */*caller*/, const itk::EventObject & /*event*/ )
+::Execute( const itk::Object * /*caller*/, const itk::EventObject & /*event*/ )
```
Comment (thewtex) [inline include/itkVariationalRegistrationStopCriterion.hxx]: An improvement for compilers and semantic code is to use the `itkNotUsed` macro.

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -14,7 +14,6 @@ set(VariationalRegistration_SRC
 
 if(WIN32) # Check if we are on Windows
   if(MSVC) # Check if we are using the Visual Studio compiler
-    message(WARNING " Module_VariationalRegistration is not testet for Visual C++ Compilers")
```
Comment (thewtex) [inline src/CMakeLists.txt:17]: yaay :+1:

**Review by thewtex — CHANGES_REQUESTED** [review]
Thanks for addressing these. A suggestion inline.

**Comment by fbudin69500** [comment]
@thewtex Added itkUnused

**Review by thewtex — APPROVED** [review]
:100:

**Pull request merged**